contributors/build-readme.py

```diff
@@ -0,0 +1,30 @@
+import json
+from common import Contributor
+import sys
+from typing import List
+from html import escape
+
+def format_contributor(contributor: Contributor) -> str:
+    return f'<td align="center"><sub><img src="{contributor.avatar_url}" width="100" height="100"><br><b>{escape(contributor.name)}</b><br>{", ".join(contributor.contributions)}</sub></td>'
```

Comment:
Is referencing the Discord CDN directly something that’s allowed?
I’m also worried a bit about the implications of always pulling icons from Discord automatically- could be exploited maliciously, potentially. Though it’s maybe not a huge concern given this is not a the main site page?

Reply:
The CDN is the correct way to get avatar images, and its the same way bots do it (I just haven't bothered with a full-blown library for a single API request): https://discord.com/developers/docs/reference#image-formatting
As for abusing the images, then the Discord TOS should prevent anything horrible, and its also only pulling from those we've granted the role to, and so we can easily remove people if needed.